symplyphysics/laws/electricity/circuits/varactor_junction_p_n_capacity.py

```diff
@@ -0,0 +1,41 @@
+from sympy import Eq, solve
+from symplyphysics import (units, Quantity, Symbol, validate_input, validate_output,
+                           dimensionless,)
+
+# Description
+## The principle of operation of the variator is based on the dependence of the barrier capacitance of the p-n junction on the voltage value.
+## Knowing the doping coefficient, the material parameter and the capacity without bias voltage, it is possible to calculate the capacity
+## of the varactor at a given voltage.
+
+## Law is: C =  C0 / (1 - V / V0)^y, where
+## C - barrier capacity of the p-n junction,
+## C0 - barrier capacitance of the p-n junction without bias voltage,
+## V - voltage,
+## V0 - material parameter,
```

Comment:
What's this?

Reply:
> What's this?

A certain constant having the dimension of the electric voltage and depending on the material.

Reply:
Add to description.